packages/clerk-js/src/core/devBrowserHandler.ts

```diff
@@ -69,6 +69,7 @@ export default function createDevBrowserHandler({
 
   // location.host != *.[lcl.dev](http://lcl.dev)
   async function setThirdPartyCookieForDevBrowser(): Promise<void> {
+    // ❓: Is proxyUrl needed here ?
```

Comment:
❓  Is proxyUrl needed here ?

Reply:
No, this affects only legacy, non-URL-Based session synced development instances.